root/user/UserProfile.js

```diff
@@ -24,6 +24,7 @@ import commaOnlyList from '../static/scripts/common/i18n/commaOnlyList';
 import {formatCount, formatPercentage} from '../statistics/utilities';
 import formatUserDate from '../utility/formatUserDate';
 import {canNominate} from '../utility/voting';
+import { returnToCurrentPage } from '../utility/returnUri';
```

Comment:
Why the padding? :)

Reply:
I use a vscode-js-import plugin to help import stuff and it adds those.